.gitmodules

```diff
@@ -45,5 +45,5 @@
 	url = https://github.com/uNetworking/uSockets.git
 	shallow = true
 [submodule "carta-backend-test-data"]
-	path = carta-backend-test-data
+	path = test/carta-backend-test-data
```

Comment:
The submodule directory should just be called `data`, like before. It doesn't make sense to use the full repository name here.

You can use `git mv` again to rename it.